include/tvm/runtime/packed_func.h

```diff
@@ -60,6 +60,12 @@ namespace tvm {
 class Integer;
 
 namespace runtime {
+
+// Datatype utilities needed at runtime. See src/runtime/custom_datatype_util.cc.
+TVM_DLL std::string GetCustomTypeName(uint8_t type_code);
```

Comment:
Should we put these utilities into a separate header to cleanly delineate the data type utility functions? Thoughts @tqchen ?

Reply:
Given that this is a single function header, it is fine